deploy/bin/backup.sh

```diff
@@ -2,31 +2,29 @@
 
 set -euxo pipefail
 
-# We are changing the backup format and where they are stored. We want to
-# retain 30 days of backups across both locations and formats. Once there
-# are none of the old format remaining, this can be updated to just refer
-# to the new location.
-REPO_ROOT="/app"
-ORIGINAL_BACKUP_DIR="/storage"
-BACKUP_DIR="$ORIGINAL_BACKUP_DIR/backup/db"
+STORAGE_ROOT="/storage"
```

Comment:
I think this should use the `DATABASE_DIR` [env  var](https://github.com/opensafely-core/opencodelists/blob/fe4bdb09469441e23a1c552fe47305012a8b9d45/DEPLOY.md?plain=1#L34) instead